app/main.py

```diff
@@ -1,3 +1,56 @@
+from typing import Union
+
```

Comment:
Consider adding a docstring to the class to describe its purpose and usage. This will improve code readability and maintainability.